basic/stam/StamExAnswer.py

```diff
@@ -0,0 +1,35 @@
+"""Reads a file, edits it and saves it in a new location."""
```

Comment:
Please make sure you name all python files like so:
* No capital letters, only small letters, underscores and (rarely) numbers
* Most meaningful name possible. In this training, you can remove usage of words like "exercise" and "answer", as it's not the essence of the code written
* Short as possible, but not too short if readability will suffer from it